source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4347,11 +4347,11 @@ export class DateInput extends Input {
         };
 
         if (this.defaultValue) {
```

Comment:
Add @adambarlow and @JeanRoca from PM side to see if there is any customer interested in this change.

Reply:
@licanhua Unfortunately, there are no customers requesting this feature that I could find.

Reply:
FYI, I found user is able to set `today` as utc, local time stamp or special time zone like this without this feature

```json
        {
            "type": "Input.Time",
            "value": "${convertFromUTC(utcNow(), 'New Zealand Standard Time', 'HH:mm')}"
        },
        {
            "id": "test",
            "type": "Input.Date",
            "value": "${convertFromUTC(utcNow(), 'New Zealand Standard Time', 'yyyy-MM-dd')}",
        },

```

```
        {
            "type": "Input.Date",
            "value": "${formatDateTime(utcNow(), 'yyyy-MM-dd')}"
        },

```

Reply:
@licanhua This is good of course, but it doesn't solve the **locale** problem. The card author needs to now the time zone information of the user to set the date what is not possible in most cases :) 
Setting the value to Today would allow us to get the local time for the user.